### ISSUE TITLE
Got about zero acc When validating resnet50_no_fusion

### ISSUE THREAD

**Issue by TimeEternal** [microsoft/multiview-human-pose-estimation-pytorch#19]
Training log is like below,  train_acc is around 1, but test_acc is 0, only used h36m dataset.
I also train the fusion model, but i got same train_acc and test acc
What happened? Is the model overfitting?
[256_nofusion_2019-12-10-09-57_train.log](https://github.com/microsoft/multiview-human-pose-estimation-pytorch/files/3943725/256_nofusion_2019-12-10-09-57_train.log)

**Comment by CHUNYUWANG** [comment]
Can you help check if the saved debug images in the ``output'' directory are correct? In particular, the images with names ``validation_''.

**Comment by CHUNYUWANG** [comment]
I tried to run the noFusion experiment. The log of the first epoch is as follows:

Test: [0/1250]  Time 26.000 (26.000)    Loss 0.0133 (0.0133)    Accuracy 1.000 (1.000)
Test: [100/1250]        Time 0.212 (0.411)      Loss 0.0334 (0.0276)    Accuracy 0.971 (0.981)
Test: [200/1250]        Time 0.102 (0.274)      Loss 0.0252 (0.0288)    Accuracy 0.996 (0.981)
Test: [300/1250]        Time 0.107 (0.225)      Loss 0.0325 (0.0298)    Accuracy 0.978 (0.978)
Test: [400/1250]        Time 0.151 (0.201)      Loss 0.0421 (0.0301)    Accuracy 0.945 (0.976)
Test: [500/1250]        Time 0.158 (0.187)      Loss 0.0127 (0.0310)    Accuracy 1.000 (0.973)
Test: [600/1250]        Time 0.155 (0.178)      Loss 0.0169 (0.0298)    Accuracy 1.000 (0.975)
Test: [700/1250]        Time 0.112 (0.170)      Loss 0.0309 (0.0293)    Accuracy 0.985 (0.976)
Test: [800/1250]        Time 0.158 (0.165)      Loss 0.0288 (0.0291)    Accuracy 0.985 (0.976)
Test: [900/1250]        Time 0.092 (0.162)      Loss 0.0306 (0.0291)    Accuracy 0.989 (0.976)
Test: [1000/1250]       Time 0.143 (0.159)      Loss 0.0290 (0.0294)    Accuracy 0.978 (0.976)
Test: [1100/1250]       Time 0.156 (0.156)      Loss 0.0329 (0.0292)    Accuracy 0.985 (0.976)
Test: [1200/1250]       Time 0.119 (0.154)      Loss 0.0250 (0.0304)    Accuracy 1.000 (0.972)
| Arch | root | rhip | rkne | rank | lhip | lkne | lank | belly | neck | nose | head | lsho | lelb | lwri | rsho | relb | rwri |
|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
| 256x256_multiview_pose_resnet_50_d256d256d256 | 1.000 | 0.983 | 0.966 | 0.910 | 0.987 | 0.947 | 0.686 | 0.990 | 1.000 | 1.000 | 1.000 | 0.917 | 0.816 | 0.776 | 0.928 | 0.797 | 0.725 |

**Comment by TimeEternal** [comment]
I think there must be some mistakes in validate code, I trained and tested the nofusion model using the same data, I changed the multiview_h36m.py  line 98 [::5] to [::64], so that train and test data must be identical, but train_acc is high test_acc is low too. 
my pytorvh version is 0.4.1, ubuntu is 4.15.0-69-generic.
Is there some difference between your local  respository and remote one?

**Comment by TimeEternal** [comment]
I reclone the respository， it works properly now. 
Thanks for your reply
